lib/Genesis/Helpers.pm

```diff
@@ -196,10 +196,52 @@ export -f bosh_cpi
 ###
 export __cloud_config_ok="yes"
 
+# Support function for cloud_config_needs static_ips
+__ip2dec() {
+	local __acc=0 IFS='.' __b __ip="$1"
+	# shellcheck disable=SC2068
+	for __b in ${__ip[@]} ; do
+		(( __acc = (__acc << 8) + __b ))
+	done
+	unset IFS
+	echo $__acc
+}
+export -f __ip2dec
+
 cloud_config_needs() {
   local __type=${1:?cloud_config_needs() - must specify a type}; shift
   local __name
 
+	# Special check for static_ips
+	if [[ "${__type}" == "static_ips" ]] ; then
+		local __network=${1:?cloud_config_needs(static_ips) - must supply network name}; shift
+		local __count=${1:?cloud_config_needs(static_ips) - must supply static_ip count} ; shift
+
+		local __ips __sum=0 __f __x __l
+		__ips=$(spruce json "$GENESIS_CLOUD_CONFIG" | \
+			jq -r --arg network "$__network" '.networks[]| select(.name == $network) | .subnets[] | .static[]')
+
+		while read -r range ; do
+			read -r __f __x __l < <(echo "$range")
```

Comment:
IP ranges in BOSH cloud-config may not have whitespace around the delimiter.

For example:

```
subnets:
 - 10.0.0.1-10.0.0.15
```

is valid, and possible.